### DISCUSSION TITLE
UDM-Pro // Keep Modem as first router

### DISCUSSION THREAD

**Discussion by hutje** [fabianishere/udm-iptv#84] [Q&A]
I need the KPN box V12 to be able to still use the telephone connection.
As Internet connection I need to setup WAN with DHCPv4 and DHCPv6. I have set-up the UDM-Pro to have DMZ.

On the UDM-Pro I want to be able to connect IPTV through the UDM-Pro, but this won't work. Watching tv-shows on-demand will work, but live tv doesn't give more than 2 seconds before freezing. Then I will get an error message.

Does anybody know how to get IPTV with still using the KPN Box v12?
Or is it possible with a few extra settings to connect the KPN Box behind the UDM-Pro and configure the VLAN and port-forward to still have the Box act as telephone station? (I don't own a VoIP system)

Thanks!

**Comment by fabianishere** [comment]
Try setting `IPTV_WAN_VLAN` to 0.

**Reply by hutje** [reply]
For the settings which I can change with `dpkg-reconfigure -p medium udm-iptv`?
And with the setup with the kpn box still as first device?

**Reply by fabianishere** [reply]
Yes, when running the configuration program, say **no** to the following question: **Is IPTV traffic carried over a separate VLAN?**

**Reply by hutje** [reply]
Can I just rerun the script or can I manually change the file with `dpkg-reconfigure -p medium udm-iptv`?

**Reply by fabianishere** [reply]
If you change `/etc/udm-iptv.conf`, you'll have to restart the service via `systemctl restart udm-iptv`. If you use `dpkg-reconfigure`, it does that automatically AFAIK.

**Reply by hutje** [reply]
Do I need to change the `IP ranges from which the IPTV traffic originates:`?
Standard is: `213.75.0.0/16,217.166.0.0/16`

The IP of my UDM-Pro is 192.168.2.8. Given from the original modem (KPN Box) with 192.168.2.254.
All local IP-addresses.

**Reply by fabianishere** [reply]
Does IPTV work with the settings I recommended?

**Reply by hutje** [reply]
No, unfortunately not. 
I did get something else. When I give my UDM-Pro in WAN-settings, VLAN 6; the UDM-Pro doesn't get an IP-address, but the IPTV apparently does work. But the router has trouble getting an IP-address  Nothing else does have internet access, but the IPTV does have live tv.

**Reply by fabianishere** [reply]
Interesting observation, possibly some firewall rules interfering?

**Reply by hutje** [reply]
Only the ones I can't change. 
![image](https://user-images.githubusercontent.com/15308350/156843742-3b4d2557-a282-459a-8608-d79073d62e6c.png)

**Comment by hutje** [comment]
IPTV did work without the modem as your tool was intended btw. The problem is that in our setup the modem is required to be able to use the original telephone cable.

**Reply by jwillemsen** [reply]
Can’t you use for example the Siemens N300 dect for the phone? That way you don’t need to use the KPN modem

**Reply by hutje** [reply]
I was thinking the same. Will consider this option. First need to be sure that the VLAN for VoIP also works... Will try a spare VoIP station from work before buying the Siemens N300.

**Reply by hutje** [reply]
Not yet tested, but would it be possible to somehow use the KPN Box as VoIP gateway? To connect the KPN Box to the UDM-Pro with LAN-port instead of WAN.
Don't know how it will act, just an idea.
